Source/URedis/Private/URedis.cpp

```diff
@@ -38,4 +38,21 @@ FString FURedis::Ping(TOptional<FString> message) const {
     return UTF8_TO_TCHAR(reply.c_str());
 }
 
+bool FURedis::Set(FStringView key, FStringView value) const {
+    check(_instance);
```

Comment:
no need to check for instance validity, after it's construction we are sure that this is a valid unique pointer

Reply:
Are you sure that we dont need validity check?
Function `Connect()` is not a constructor, so User just may skip or forget about it and then it will crush because of the null pointer function calling.

Reply:
That's a fair point, but the check doesn't prevent the program from crashing. In any case, all our methods - except from `Connect - require a valid `_instance`. 

In short `check(_instance)` and `_instance->whatever` will make the program crash the same way (note: moreover, in shipping builds checks are stripped out from the source code). Thus, instead of adding a `check(_instance)` at the beginning of all the methods, we can just assume it to be mandatory valid. If this is not true (for whatever reason, a failed connection or a missing call to `Connect`) the program is invalid and it will crash anyway

Reply:
So, Instead of check functions I added validity check. Let me know your thoughts.

Reply:
The validity check is useless because it hides the real problem (invalid instance - no connection/connection died). You can remove it. If it crashes, it's because there's no connection, and that's ok IMO. This way we don't hide the problem from the caller.

Perfect for the documentation